Dockerfile

```diff
@@ -0,0 +1,100 @@
+FROM ubuntu:14.04
+MAINTAINER Bilal Salam <bilal.salam@kitware.com>
+
+
+RUN apt-get update && \
+apt-get install -y \
+git \
+wget \
+python-qt4 \
+openslide-tools python-openslide \
+build-essential \
+swig \
+make \
+zlib1g-dev \
+curl \
+libcurl4-openssl-dev \
+libexpat1-dev \
+unzip \
+libhdf5-dev \
+libjpeg-dev \
+libopenslide-dev \
+libpng12-dev \
+libpython3-dev \
+libtiff5-dev \
+cmake \
+openslide-tools && \
+apt-get autoremove && \
+rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+WORKDIR /
+
+# Install miniconda
+ENV build_path=$PWD/build
+RUN mkdir -p $build_path && \
+    wget https://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh \
+    -O $build_path/install_miniconda.sh && \
+    bash $build_path/install_miniconda.sh -b -p $build_path/miniconda && \
+    rm $build_path/install_miniconda.sh && \
+    chmod -R +r $build_path && \
+    chmod +x $build_path/miniconda/bin/python
+ENV PATH=$build_path/miniconda/bin:${PATH}
+
+
+ENV htk_path=$PWD/HistomicsTK
+RUN mkdir -p $htk_path
```

Comment:
Lets not do this here. Let HistomicsTK take care of creating a directory for itself.